requirements_all.txt

```diff
@@ -1251,6 +1251,12 @@ mutagen==1.47.0
 # homeassistant.components.mutesync
 mutesync==0.0.1
 
+# homeassistant.components.mycroft
+mycroftapi==2.0
```

Comment:
A rebase seems to have gone wrong. This should be removed. This is why CI is failing.

Reply:
@MartinHjelmare it fixed some CI errors, but something else came up. Again, seems related to another integration?

Reply:
Just rebase on latest dev branch. We fixed that yesterday.